jax/interpreters/pxla.py

```diff
@@ -1703,6 +1757,7 @@ def __init__(self,
                mesh: Mesh,
                local_in_untiled_avals: Sequence[ShapedArray],
                local_out_untiled_avals: Sequence[ShapedArray],
+               global_out_avals: Optional[Sequence[ShapedArray]],
```

Comment:
Global avals should be computable form local avals, so you don't need this parameter.